parsers/python.c

```diff
@@ -157,6 +158,7 @@ static const keywordTable PythonKeywordTable[] = {
 	{ "lambda",			KEYWORD_lambda			},
 	{ "pass",			KEYWORD_pass			},
 	{ "return",			KEYWORD_return			},
+	{ "type",			KEYWORD_type			},  /* not a real keyword */
```

Comment:
This is not workable: it breaks `type = something` assignment (caught by the test suite, good work!), and might break other valid Python code such as `def type(...):` and so on.